tools/build-docs

```diff
@@ -1,3 +1,26 @@
 #!/usr/bin/env bash
 set -eu
-make -C "$(dirname "$0")"/../docs clean html
+
+args="$(getopt -o c --long clean -- "$@")"
+eval "set -- $args"
+clean=
+while true; do
+    case "$1" in
+        -c | --clean)
+            clean=1
+            shift
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *) exit 1 ;;
+    esac
+done
+
+if [ -n "$clean" ]; then
+    make -C "$(dirname "$0")"/../docs clean html
+else
+    echo "Warning: tools/build-docs --clean is necessary for the navigation/left sidebar to update. "
```

Comment:
Let's put this output after the below, and use terminal color codes to colorize it.

(It's possible you should just rewrite this in Python, since then you can use e.g. `WARNING` from `scripts/lib/zulip_tools.py` rather than doing color codes by hand -- I think `argparse` is cleaner than `getopt` and this will be useful for letting us do the `sanity_check` as well).